pymongo_auth_aws/auth.py

```diff
@@ -60,6 +103,7 @@ def _aws_temp_credentials():
             raise PyMongoAuthAwsError(
                 'temporary MONGODB-AWS credentials could not be obtained, '
                 'error: %s' % (exc,))
+
```

Comment:
Does the _irsa_assume_role return need to be cached too?

Reply:
Done